usagov/usagov-2021
# Usagov 1686 give images explicit dims

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1686

## Description

Give elements explicit height and width dimensions

Added height to USAgov logo, banner flag and icons, carousel prev and next buttons, and jump down jump button.
Maintains responsiveness.

## Type of Changes

- [x ] New Feature
  - Add screenshot showing what it should look like

## Testing Instructions



localhost > chrome dev tools > run lighthouse analytics > check passed audits to see "Image elements have explicit width and height"

Increase and decrease the size of the viewport to make sure the USAgov logo reacts responsively and other icons do not. Make sure it matches the behavior of the prod site.

### Requires New Config
- [ x] No

### Requires New Content
- [ x] No

### Validation Steps
- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
